pkgs/applications/misc/openlp/BUGS

```diff
@@ -0,0 +1,8 @@
+1. checks must be disabled because they are lacking the qt env.
+   (see ./lib.nix and https://discourse.nixos.org/t/qt-plugin-path-unset-in-test-phase/)
+1. segfault on exit. Not a real problem, according to debug log, everything
+   shuts down correctly. Maybe related to https://forums.openlp.org/discussion/3620/crash-on-exit.
+   Plan: Wait for OpenLP-3, since it is already in alpha 2
+   (2020-06-22; news: https://openlp.org/blog/)
+1. {my,postgre}sql and jenkins are disabled, because probably almost no one uses it and I
+   (jorsn) didn't test it. If someone needs support, please contact me.
```

Comment:
why is this relevant to nixpkgs?

Reply:
They are issues with the packaging. As this is a relatively complex package, I thought it would be nice to write down them publicly instead of letting someone stumble over them and wonder. Also, reviewers are informed about me submitting this despite the issues. Normally I wouldn't submit anything which segfaults, but in this case I consider this a minor issue. At least they serve as a reminder for me try to resolve them (1 and 2) when I package the next release. Then, there is the link to the relevant discoure thread.

Do you think these notes are irrelevant to nixpkgs or should be just outsourced? Then they would be either a GH issues or private.

Reply:
I guess my concern is that people are likely to never become aware of this

Reply:
If you like I can remove the note and keep it private. I have no great affections to this.

Reply:
I agree with Jon, I don't think users will ever find this file. It's better to move this to a comment inside the main Nix expression, which can be more easily viewed with `nix edit nixpkgs.openlp`. Another option could be to put this inside a meta attribute: there isn't an appropriate one yet but I checked and Nix seems to allow arbitrary attributes:

```$ nix-env -f. -qa -A hello --json | jq '.hello.meta.bugs' --raw-output
This a list of bugs which affect this software packaged
in Nixpkgs:
  - blabla.
  - blablabla.
```

Reply:
Ok: I moved the bugs into a `meta.bugs` attr and moved meta into `lib.nix`, so that people using `nix edit` land directly in the source file.

Reply:
ofborg complained about `meta.bugs`, so I moved it in `meta.longDescription`.